### ISSUE TITLE
Translation among annotations

### ISSUE THREAD

**Issue by MiqG** [nuno-agostinho/psichomics#420]
Hi,

thank you for creating such a great package that unifies the field of alternative splicing analysis. 

Currently, I am trying to translate the event annotations from [Thomas *et al.* 2020](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6962552/) into VastDB identifiers. So far, I have been able to spot that they used MISO annotations from hg19, but I cannot figure out which event they map to and with which logic. I thought that since you are working at the intersection of all the AS tools, this would be a piece of cake!

For example:
- event in Thomas 2020: 
```
se@1:107963765:107973372:+|1:107963765:107979287:+
```
- possible mappings in VastDB annotation from hg19 searching every coordinate separately:
```
GENE    EVENT   COORD_o LE_o    FULL_CO
NTNG1    HsaEX0044072    chr1:107963706-107963765   60    chr1:107961267,107963706-107963765,108023233
NTNG1    HsaEX0044073    chr1:107973372-107973539    168    chr1:107950330,107973372-107973539,108023233
NTNG1    HsaEX0044074   chr1:107979287-107979421   135    chr1:107973539,107979287-107979421,108023233
```

Thanks a lot in advance for your help!

Miquel

**Comment by nuno-agostinho** [comment]
Hey Miquel!

> `se@1:107963765:107973372:+|1:107963765:107979287:+`

I don't think that identifier is from MISO...

Anyway, you can find in Bioconductor an [alternativeSplicingEvents.hg19 annotation](http://bioconductor.org/packages/alternativeSplicingEvents.hg19) that I created with the conversion between MISO and VAST-TOOLS identifiers for multiple events (probably not all though).

Please note that:
- MISO IDs are in the following format (as according to [MISO documentation](https://miso.readthedocs.io/en/fastmiso/#gff-based-alternative-events-format)): `chr1:4775654:4775821:-@chr1:4774032:4774186:-@chr1:4772649:4772814:-`
- VAST-TOOLS has two different identifiers per event (e.g. `HsaEX0000001` and `ENSG00000254427_CASSETTE1` correspond to the same identifier). I used the `ENSG00000254427_CASSETTE1` format in the annotation table, but you can easily convert those to the `HsaEX` identifiers by downloading the [human VASTDB annotation](https://github.com/vastgroup/vast-tools#vastdb-libraries) and going to the file `FILES/New_ID-Hsa.txt.gz` where you have the conversion between the different VAST-TOOLS identifiers.

If you need further help, feel free to ask me! :)

**Comment by MiqG** [comment]
Thanks a lot for the quick help! I'll contact the authors then :)

**Comment by nuno-agostinho** [comment]
Hey Miguel, any news on this? Anything that I can help you with? :)

**Comment by MiqG** [comment]
Hi Nuno, yes I got back from them! 

They explained to me that their exon definition was: `first junction|exclusion junction`.  For example, in `se@1:107963765:107973372:+|1:107963765:107979287:+` the first junction goes from 107963765 to 107973372, and when the exon is skipped the junction goes from the second portion of coordinates from 107963765 to 107979287; that's why the first coordinate is the same for both junctions.
With this, I was able to use the coordinates to map their events to VastDB identifiers using the REF_CO column of the hg19 annotation :)

Cheers!
